app/code/VConnect/Blog/Setup/Patch/Data/AddTestPostRecordToTable.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace VConnect\Blog\Setup\Patch\Data;
+
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+use VConnect\Blog\Api\Data\PostInterface;
+
+class AddTestPostRecordToTable implements DataPatchInterface
+{
+    private ModuleDataSetupInterface $moduleDataSetup;
+
+    public function __construct(ModuleDataSetupInterface $moduleDataSetup)
+    {
+        $this->moduleDataSetup = $moduleDataSetup;
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public function apply(): DataPatchInterface
+    {
+        $tableName = $this->moduleDataSetup->getTable(PostInterface::MAIN_TABLE);
```

Comment:
i think you should have used a resource model at least in order to save one demo post here instead of that direct query